README.rst

```diff
@@ -117,15 +117,15 @@ You can immediately use Ivy to train a neural network, using your favorite frame
             x = ivy.relu(self.linear0(x))
             return ivy.sigmoid(self.linear1(x))
 
-    ivy.set_backend('torch')  # change to any backend!
+    ivy.set_framework('torch')  # change to any framework!
```

Comment:
`set_framework` is used in the pypi version `Ivy v1.1.9` and as of now we are using the development version we use `set_backend`

Reply:
Thanks for the review @hirwa-nshuti, i now understand thought there were inconsistencies in the documentation.